src/components/scrollbar/ScrollBar.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactNode } from 'react';
+import Scrollbars from 'react-custom-scrollbars-2';
+
+type ScrollBarProps = {
+  className?: string;
+  children?: ReactNode;
+};
+
+const ScrollBar = ({
+  children,
+  className = '',
+}: ScrollBarProps): JSX.Element => (
+  <Scrollbars
+    className={className}
+    renderTrackHorizontal={() => <div style={{ display: 'none' }} />}
+    renderThumbVertical={({ style, ...props }) => (
+      <div
+        // eslint-disable-next-line react/jsx-props-no-spreading

```

Comment:
can we add this to global ESLint configuration? I've disabled this rule a couple of times already as well, I don't see a reason to keep it.